### DISCUSSION TITLE
Cancellation of ABO/ Kündigung ABO

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#661] [Q&A] [answered]
Cancellation of ABO 

Hello everyone, can I continue to use the Cruiser app until the end of the term or does the user authorization expire directly with the cancellation? I would like to cancel both Cruiser licenses now as a precaution so that they are not inadvertently extended. Both licenses will not be updated to V5 and when Android 15 comes, V4 will not work either. If 3D goes to V5 at some point, you can get back in.

Kündigung ABO 

Hallo zusammen, kann ich die Cruiser App bis zum Ende der Laufzeit weiter nutzen oder erlischt mit der Kündigung auch direkt die Nutzungsberechtigung? Ich möchte jetzt schon einmal vorsorglich beide Cruiser Lizenzen kündigen, damit sie  nicht versehentlich verlängert werden. Beide Lizenzen werden nicht auf V5 upgedatet und wenn Android 15 kommt, soll V4 ja  auch nicht mer gehen. Sollte irgendwann 3D in V5 gehen, kann man ja wieder einsteigen.

**Comment by devemux86** [comment, marked as answer]
Google Play manages the subscriptions. From [Google Play Help](https://support.google.com/googleplay/answer/7018481?sjid=11043875566425791218-EU&co=GENIE.Platform%3DAndroid&oco=1&hl=en):

What happens after you cancel?

When you cancel a subscription you’ll still be able to use your subscription for the time you’ve already paid.

For example, if you buy a one-year subscription on January 1 for $10 and decide to cancel your subscription on July 1:
- You’ll have access to the subscription until December 31.
- You won’t be charged another yearly subscription of $10 the following January 1.

---

Google Play verwaltet die Abonnements. Aus der [Google Play-Hilfe](https://support.google.com/googleplay/answer/7018481?sjid=11043875566425791218-EU&co=GENIE.Platform%3DAndroid&oco=1&hl=de):

Was geschieht nach der Kündigung?

Wenn Sie ein Abo kündigen, können Sie es für den bereits bezahlten Zeitraum weiterhin nutzen.

Beispiel: Sie kaufen am 1. Januar ein Jahresabo für 10 € und kündigen es am 1. Juli:
- Sie können das Abo dann noch bis zum 31. Dezember nutzen.
- Ihnen werden zum folgenden 1. Januar keine weiteren 10 € für ein weiteres Jahresabo berechnet.

**Comment by ToToGSARider** [comment]
Okay, Danke für die Antwort. Kündigung wurde von Google zum 25.06.2025 bestätigt. 

Okay, thanks for the answer. Termination has been confirmed by Google as of 25.06.2025.

**Reply by ToToGSARider** [reply]
Great, the second one also worked without any problems.

Super, beim zweiten hat es auch ohne Probleme geklappt.
